preprocessing/preprocess_test.py

```diff
@@ -0,0 +1,10 @@
+import pytest as pytest
+
+import preprocess
```

Comment:
This file is being picked up by pytest, and is throwing an error (hence the failing Travis CI build). Use `import preprocessing/preprocess` instead if we want to run it with pytest; otherwise rename the _test suffix